student-os/scripts/repair_import_review.py

```
@@ -0,0 +1,516 @@
+        )
+
+    governed = mark_auto_repaired(replacement, needs_review=True)
+    risk_items = diagnose_import_risks(governed)
+    for risk in risk_items:
+        code = str(risk.get("code", ""))
+        severity = "error" if code in {"math-dollar-unbalanced", "latex-left-right-unbalanced"} else "warning"
+        issues.append(issue(f"remaining-{code}", "Replacement still has import risk diagnostics.", severity=severity, detail=risk))
+
+    original_text = ""
+    if resolved_target and case_state_root is not None:
+        if resolved_target.suffix.lower() != ".md" or not is_relative_to(resolved_target, case_state_root):
```

Ruff (0.16.2)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)